detox/android/detox/src/main/java/com/wix/detox/espresso/DetoxMatcher.java

```diff
@@ -75,6 +77,10 @@ private DetoxMatcher() {
         return isDisplayingAtLeast(75);
     }
 
+    public static Matcher<View> matcherForMinimumVisiblePercent(double percentage) {
```

Comment:
I really think we should stick with `matcherForSufficientlyVisible()` in this context, so as to stay coherent with iOS (EarlGrey). In any case, no need for both methods...